src/styles/button.css

```diff
@@ -1 +1,68 @@
 /* button */
+
+.ui.button {
+  background-color: var(--primary-pms-362_900);
+  color: var(--shade-white);
+  font-family: var(--gotham);
+  border-radius: var(--border-radius-3);
+}
+
+/** Focus **/
+.ui.button:focus {
+  background-color: var(--primary-pms-362_900);
+  color: var(--shade-white);
+}
+
+/** Hover **/
+
+.ui.button:hover {
+  background-color: var(--primary-pms-362_800);
+  color: var(--shade-white);
+}
+
+/** Active **/
+
+.ui.button:active {
+  background-color: var(--primary-pms-362_700);
+  color: var(--shade-white);
+}
+
+/** Secondary Button **/
+
+.ui.secondary.button {
+  background-color: var(--shade-white);
+  color: var(--primary-pms-362_900);
+  border: 1px solid var(--primary-pms-362_900);
+  border-radius: var(--border-radius-3);
+  font-family: var(--gotham);
```

Comment:
It's already in `.ui.button`

Reply:
Nice! I will remove the `font-family` and `border-radius`